package.json

```
@@ -76,6 +78,10 @@
         "yup": "^1.4.0"
     },
     "devDependencies": {
+        "@testing-library/jest-dom": "^6.6.3",
```

Biome (1.9.4)

[error] The key @testing-library/jest-dom was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)